src/components/form/DeleteFileForm.vue

```diff
@@ -60,11 +60,12 @@ const submitting = ref(false);
  */
 function onSubmit() {
   if (isFolderWithChildren.value && !confirmDelete.value) {
-    return Notify.create({
+    Notify.create({
       type: 'negative',
       message: t('errors.fileExplorer.folder.delete'),
       html: true,
     });
+    return Promise.resolve();
```

Comment:
`return;`

Modify `return deleteProjectFile` by `deleteProjectFile`

Reply:
Done.

Reply:
Also changed the `onSubmit` function type to `async` in order to make the unit tests pass